collection_json.py

```diff
@@ -270,7 +293,7 @@ def __getattr__(self, name):
 
     @property
     def properties(self):
-        """Return a list of names that can be looked up on the template."""
+        """Return a list of names that can be looked up on the item."""
```

Comment:
why this rename? This function operates on the template object, not the item object.